### PR TITLE
Add remoteAddress logging to request event

### PR DESCRIPTION
**Pull request by jasisk**

### CODE REVIEW

File: lib/series.js [hunk 1]
```diff
@@ -71,7 +71,10 @@ module.exports = {
             return String(eventData.labels);
         },
         host: Os.hostname,
-        pid: 'pid'
+        pid: 'pid',
+        remoteIp: function (eventData) {
+            return String(eventData.source && eventData.source.remoteAddress);
```
Comment (totherik) [inline lib/series.js]: Is the intent to have the string `'undefined'` in the dataset?

Comment (jasisk) [inline lib/series.js]: Yeah but that's only because I wasn't sure if influx over http supports null.

Comment (totherik) [inline lib/series.js]: It just gets serialized to json, right? So undefined properties are omitted during `JSON.stringify` and (I assume) influx is none the wiser.

Comment (jasisk) [inline lib/series.js]: Sounds good to me.

**Comment by totherik** [comment]
:+1:

**Pull request merged**